### PR TITLE
[6.x.x] Backport build scripts

### PR DESCRIPTION
**Pull request by reinhapa**
Backport `develop` Github actions

### CODE REVIEW

File: exist-parent/pom.xml [hunk 1]
```diff
@@ -563,11 +563,10 @@
                 <plugin>
                     <groupId>org.owasp</groupId>
                     <artifactId>dependency-check-maven</artifactId>
-                    <version>8.0.2</version>
+                    <version>8.3.1</version>
```
Comment (dizzzz) [inline exist-parent/pom.xml:566]: please update to 8.4.2

**Comment by reinhapa** [comment]
@dizzzz just squashed all changes together. Ready to merge.

**Review by dizzzz — APPROVED** [review]

**Comment by adamretter** [comment]
@reinhapa I am not sure why but it seems to show that none of the CI is running for this, could you take a look please?

**Comment by reinhapa** [comment]
> @reinhapa I am not sure why but it seems to show that none of the CI is running for this, could you take a look please?

@adamretter I have no idea. It went into this state, when I squashed the commits together...

**Comment by dizzzz** [comment]
image

**Review by line-o — APPROVED** [review]
Thank you very much @reinhapa
Now we will have to monitor if the windows build is really broken at the moment

**Pull request merged**